iceoryx2-ffi/cxx/tests/src/service_publish_subscribe_tests.cpp

```diff
@@ -968,6 +981,20 @@ TYPED_TEST(ServicePublishSubscribeTest,
     ASSERT_TRUE(sut_open.has_error());
 }
 
+TYPED_TEST(ServicePublishSubscribeTest,
```

Comment:
Can you split this into two tests.
1. Same payload type name and same size but different alignment
2. Same payload type name and same alignment but different size

Reply:
The same goes also for the header tests. When this is done, this is ready to be merged.